### DISCUSSION TITLE
failed to find a valid digest in the 'integrity' attribute

### DISCUSSION THREAD

**Discussion by Oui-Dev** [Baroshem/nuxt-security#367] [Q&A]
Hi everyone,

I recently deployed my SSR Nuxt project (3.10.0) using nuxt-security (1.1.0), and i experienced the same issue as https://github.com/Baroshem/nuxt-security/issues/297 and https://github.com/Baroshem/nuxt-security/issues/331. It's hosted on ubuntu 20.04 vps, with nginx and Cloudflare.
I tried to disable sri with `sri: false` and it worked, but is it a security concern ?

**Comment by vejja** [comment]
Hi @Oui-Dev 
You won’t have Subresource Integrity with sri: false.
Depending on your security requirements this might be ok for you, but I can’t tell if this is the case.

Did you disable post-build processing in your Cloudflare setup?

It is hard for us to understand where the issue with Cloudflare is coming from. If you can share some more details with us this would be extremely helpful !

**Comment by Oui-Dev** [comment]
I disabled the JavaScript minification but I m keep using Brotli compression and it work flawlessly, i have also purged my cache.
So I guess the culprit was this minification. I will continue to monitor to see if the problem come back.

![Capture d'écran 2024-02-03 232754](https://github.com/Baroshem/nuxt-security/assets/70695580/dfa48071-db8e-4973-9823-b4de08f24ebf)
![Capture d'écran 2024-02-03 232803](https://github.com/Baroshem/nuxt-security/assets/70695580/d2456777-8a19-4933-9497-2d02b4710342)

Thks for your quick response 🙂
